### PR TITLE
Hestia: Mechs Are Scuffed This Fixes It Hopefully Edition.

### PR DESCRIPTION
**Pull request by Dark1Star3s**
## Changes:
Robotics can now build ballistic weaponry for exo-suits. There was a gramatical error within the "TECH_MATERIAL".
Mech health has been adjusted following live testing. This should go well with Purples armor changes.

## Did You Test It? Yes. And weapons are now in the right category.

### CODE REVIEW

**Comment by Anton-Kr** [comment]
What category do they appear in? I'd really appreciate that issue being fixed before this is merged. I'd just look at how the design datum is different from other mounted weapons and go from there.

**Comment by Dark1Star3s** [comment]
> What category do they appear in? I'd really appreciate that issue being fixed before this is merged. I'd just look at how the design datum is different from other mounted weapons and go from there.

They appear in the category where all the mech body parts do. For some reason- after looking through them I couldn't see where it's decided on where they appear. And I've compared it to other parts.

File: code/modules/research/designs/designs_mechfab.dm [hunk 1]
```diff
@@ -548,21 +548,21 @@
 /datum/design/item/mechfab/exosuit/smg
```
Comment (Anton-Kr) [inline code/modules/research/designs/designs_mechfab.dm]:
```suggestion
/datum/design/item/exosuit/smg
```
Try this and see if it works.

Comment (Dark1Star3s) [reply code/modules/research/designs/designs_mechfab.dm]: That worked, cheers.

**Comment by Dark1Star3s** [comment]
This has been updated with new mech changes.

**Comment by Dark1Star3s** [comment]
Powerloader parts now stronger than light mech parts.

**Comment by Anton-Kr** [comment]
muh anatomic PRs

**Review by Anton-Kr — APPROVED** [review]

**Pull request merged**